package.json

```diff
@@ -59,19 +85,22 @@
 			},
 			{
 				"command": "camelk.integrations.remove",
-				"title": "Remove Integration"
+				"title": "Remove Camel-K Integration"
+			},
+			{
+				"command": "camelk.integrations.log",
+				"title": "View log for Camel-K Integration"
+			},
+			{
+				"command": "camel.integrations.startproxy",
+				"title": "Camel-K: Start the kubectl proxy server"
```

Comment:
provided a follow-up enhancement https://github.com/camel-tooling/vscode-camelk/issues/38 to guide user